notary-commons/src/main/kotlin/com/d3/commons/service/FInalizationDetails.kt

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright D3 Ledger, Inc. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.d3.commons.service
+
+import com.google.gson.Gson
+import java.math.BigDecimal
+
+private val gson = Gson()
```

Comment:
there are quite many `Gson` objects among classes of the `notary-commons`. Could you think of reusing a single one if possible (maybe not for all of the classes)